DigitalExcellence/dex-frontend
# Feature/actions

## Description

Added Docker support for the frontend, as well as the NGINX configuration. Also created a workflow that now includes linting, and I solved all the linting issues as well.

## Type of change


## Checklist


## Steps to Test or Reproduce

Outline the steps to test or reproduce the PR here.
These steps will be used during release testing.

1. Create a pull request
2. Go to the actions, you should see a new action running

## Link to issue

Closes: #143
